services/php-fpm-nginx/manifest

```diff
@@ -0,0 +1,11 @@
+# Override getTags() to always point to the php image, not an invalid
+# php-fpm-nginx image.
```

Comment:
I'm not following this. I'm guessing that `NAME` can't be specified in this file itself or that creates other issues? But I don't know what those issues are by this comment.

Reply:
@deviantintegral can you weigh in on this and potentially update your comment here with the reasoning for the approach in this manifest?

Reply:
If I remember right, the issue was that it would try to build from a base image called `php-fpm-nginx`. The existing scripts assume that the directory name in `services/<service>` match the upstream image name. There is no docker hub image called `php-fpm-nginx` so this was failing out.

Am I right this is needed? If there's a better way that's fine by me.